test/xref2/github_issue_932.t/run.t

```diff
@@ -0,0 +1,67 @@
+A quick test to repro the issue found in #941
+
+  $ ocamlc -bin-annot -c foo.mli
+
+  $ odoc compile foo.cmti
+  $ odoc link foo.odoc
+
+  $ odoc html-generate --indent -o html/ foo.odocl
+
+The rendered html
+
+  $ cat html/Foo/index.html | grep "extension" -A 3
+      <div class="spec type extension anchored" id="extension-decl-A">
+       <a href="#extension-decl-A" class="anchor"></a>
+       <code>
+        <span><span class="keyword">type</span> <a href="#type-t">t</a> += 
+        </span>
+  --
+        <li id="extension-A" class="def extension anchored">
+         <a href="#extension-A" class="anchor"></a>
+         <code><span>| </span><span><span class="extension">A</span></span>
+         </code>
+        </li>
+        <li id="extension-B" class="def extension anchored">
+         <a href="#extension-B" class="anchor"></a>
+         <code><span>| </span><span><span class="extension">B</span></span>
+         </code>
+        </li>
+       </ol>
+  --
+      <div class="spec type extension anchored" id="extension-decl-C">
+       <a href="#extension-decl-C" class="anchor"></a>
+       <code>
+        <span><span class="keyword">type</span> 
+         <a href="M/index.html#type-t">M.t</a> += 
+  --
+        <li id="extension-C" class="def extension anchored">
+         <a href="#extension-C" class="anchor"></a>
+         <code><span>| </span><span><span class="extension">C</span></span>
+         </code>
+        </li>
+       </ol>
+  --
+      <li>extension-decl-A : <a href="#extension-decl-A"><code>Foo.A</code></a>
+      </li>
+      <li>extension-decl-B : <a href="#extension-decl-B"><code>Foo.B</code></a>
```

Comment:
@panglesd I managed to get the right label for the link but the url is now wrong (unless we can add multiple anchors to the type declaration).